src/components/pages/TabelaPosts.js

```diff
@@ -133,35 +135,126 @@ class TabelaPosts extends React.Component {
       });
     }
 
-    finalizados(){
-      this.setState({
-        tableData: this.state.posts.filter((e)=>{return e.status === 'Revisado'})
-      });
+    filterStatus(e,type){
+      const array = this.state.filters;
+      const index = array.indexOf(type);
+      if (e.target.checked === true){
+        array.push(type);
+        this.setState({
+          tableData: this.state.posts.filter((e) => {return e.status === type;}),
+          filters: array
+        });
+      } else {
+        if ( index > -1){
+          this.state.filters.slice(index, 1);
+        }
+        const removed = array.splice(array.indexOf(type),1);
+        this.setState({
+          tableData: this.state.posts.filter((e) => {return e.status === type;})
+        });
+      }
+      this.filtrar();
     }
 
-    emAndamento(){
-      this.setState({
-        tableData: this.state.posts.filter((e)=>{return e.status === 'Em andamento'})
-      });
+    filterCategory(e, type){
```

Comment:
Codacy found an issue: [Method 'filterCategory' has a complexity of 6. Maximum allowed is 4.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6345346)